apps/coursition/src/app/actions.ts

```diff
@@ -23,8 +29,7 @@ export const generateCheckout = async () => {
 }
 
 export const createUser = async ({ email, password }: { email: string; password: string }) => {
-  const saltOrRounds = 10
-  const passwordHash = bcrypt.hashSync(password, saltOrRounds)
+  const passwordHash = bcrypt.hashSync(password, HASH_SALT_OR_ROUNDS)
   return await prisma.user
```

Comment:
**nitpick:** still awaiting in return and also catching. Better to use `async/await` always. It would be nitpick, but I see it's not changed code, but the old one.